thepantry/admin.py

```diff
@@ -1,66 +1,41 @@
-from food import app, db
+from food import app
 from flask import render_template, send_from_directory, redirect, url_for, flash
-from flask_login import current_user, login_user, login_required, logout_user
-from models import User
-from thepantry.forms import LoginForm, ChangePasswordForm
+from flask_oidc import OpenIDConnect
 
+import config
 
-@app.login_manager.unauthorized_handler
-def unauthorized():
```

Comment:
Why was the unauthorised handler removed? How does the application react when an unauthorised user tries to make a request?

Reply:
Understandable, resolved